cli-tests/tests/strapi/version/version.test.js

```diff
@@ -0,0 +1,14 @@
+describe('--version', () => {
+  const instances = [];
+
+  beforeAll(async () => {
+    // TODO: get the test instances from helpers
+  });
+  beforeEach(async () => {
+    // TODO: reset the test instances
+  });
+
+  // TODO: run the cli command `strapi --version` and check exit code and output
+  // Should we use coffee (https://github.com/node-modules/coffee) or CLET (https://github.com/node-modules/clet) ?
+  it.todo('should output version');
```

Comment:
Just a reminder for everyone following the PR: this will be addressed in the next iterations of the initiative.